commonv5/ts/helpers/__tests__/request-test.ts

```diff
@@ -1,35 +0,0 @@
-import Endpoint, { EndpointType } from "../../sonarqube/Endpoint";
-import { getJSON } from "../request";
-
-const mockCallback = jest.fn((_options, cb) => cb(null, { name: "a" }, null));
-
-jest.mock("request", () => ({
-  get: (options, cb) => mockCallback(options, cb),
-}));
-
-describe("request", () => {
```

Comment:
Can't we keep this test?

Reply:
This is not testing anything. It tests that the "timeout" property is set, which is set with no condition in the code.

And with axios it's super hard to test that.